core/config/mainnet.json

```diff
@@ -26774,6 +26774,18 @@
                         }
                     }
                 ]
+            },
+            {
+                "name": "Paradise",
```

Comment:
I don't like words Paradise and Bomb used together here, because it's well known combination from Islamic Jihad. What people will think? Let's use just "Diffuse Difficulty Bomb" instead

Reply:
yikes, i had no idea...